icons/artifacthub.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Artifact Hub</title><path d="M12.067 0a3.055 3.055 0 00-1.59.407l-7.84 4.477A3.05 3.05 0 001.114 7.52v8.951a3.077 3.077 0 001.532 2.644l7.832 4.476a3.047 3.047 0 003.046 0l7.84-4.476a3.059 3.059 0 001.524-2.637V7.52a3.077 3.077 0 00-1.532-2.642L13.523.407a3.055 3.055 0 00-1.456-.405zM12 2.6a1.233 1.233 0 01.649.168l7.096 4.056a1.3 1.3 0 01.647 1.119v8.112a1.297 1.297 0 01-.64 1.118l-7.105 4.058a1.307 1.307 0 01-.65.173 1.283 1.283 0 01-.647-.174l-7.092-4.056a1.3 1.3 0 01-.646-1.119V7.941a1.297 1.297 0 01.64-1.118l7.105-4.052A1.294 1.294 0 0112 2.598z"/></svg>
```

Comment:
When comparing to the sorce SVG, I see some differences. Ex.
![image](https://user-images.githubusercontent.com/12817388/129728887-bbad403c-34ea-49a5-a646-651da76e6958.png)

It might be due to the SVGO precision. Maybe you will need to try with a precision of 4. It might be due to the stroke to path conversion too. Can you take a look at it and gave this another try?

Reply:
@service-paradis I did some changes in the logo and also used a precision of 4 in SVGO ... it looks better now even it's not 100% match.

Reply:
Thank you for your work. As you said, it is much better!

Tools are not really good to convert stroke to path and this icon id more difficult than it appear!
Still, we really try to have a perfect fit with the original and there are some differences that we can ignore.

I gave it a try and got the following path. It might need some adjustments to have a better fit, to optimize and to pass the linter, but you can use it as a starting point if you think it can help you.

```
m 11.999414,24.000438 c -0.616974,0.0012 -1.24203,-0.172158 -1.78,-0.5 L 2.71,19.208 C 1.6924162,18.591195 1.0700559,17.490364 1.0694135,16.300438 V 7.7304382 c 0,-1.253 0.6369062,-2.369375 1.76,-3.01 L 10.25,0.48 c 1.079982,-0.63864911 2.415018,-0.63764911 3.495,0.001 l 7.419414,4.2374382 c 1.080027,0.5961014 1.771323,1.7340139 1.762586,3.01 V 16.31 c 0,0.967687 -0.443367,2.163782 -1.636586,2.898438 l -7.510414,4.292 c -0.563437,0.343937 -1.198555,0.5 -1.781586,0.5 z m -0.5,-21.3459998 L 3.955,6.963 C 3.7544063,7.1196875 3.5759541,7.3321221 3.576,7.7304382 L 3.577,16.400438 C 3.6145313,16.620875 3.6959063,16.8285 3.95,17.035 l 7.559414,4.316438 c 0.266,0.172 0.664,0.21 0.985,0 L 20.012,17.055 c 0.231719,-0.148438 0.371,-0.388562 0.41,-0.653562 V 7.7304382 C 20.4315,7.431606 20.286058,7.1418659 20.045,6.965 L 12.499414,2.6534382 c -0.390562,-0.2393063 -0.759172,-0.1557376 -1,10e-4 z
```

Reply:
@service-paradis this looks great, I've updated the file after small adjustments (set the height to 24px and set precision to 5)
Thanks for your help :bow: